### ISSUE TITLE
UndeliverableException in FallbackReverseGeocodeObservable 

### ISSUE THREAD

**Issue by niqo01** [mcharmas/Android-ReactiveLocation#169]
See bug details including stacktrace here: http://crashes.to/s/18f5d0e0917

I am not too familiar with this library code but I think the following code:
https://github.com/mcharmas/Android-ReactiveLocation/blob/master/android-reactive-location/src/main/java/pl/charmas/android/reactivelocation2/observables/geocode/FallbackReverseGeocodeObservable.java#L43
```
    public void subscribe(ObservableEmitter<List<Address>> emitter) throws Exception {
        try {
            emitter.onNext(alternativeReverseGeocodeQuery());
            emitter.onComplete();
        } catch (Exception ex) {
            emitter.onError(ex);
        }
    }
```
Should probably be something like :
```
    public void subscribe(ObservableEmitter<List<Address>> emitter) throws Exception {
        try {
            List<Address>  lists = alternativeReverseGeocodeQuery()
            if (!emitter.isDisposed()) {
                 emitter.onNext(lists);
                 emitter.onComplete();
           }
        } catch (Exception ex) {
           if (!emitter.isDisposed()) emitter.onError(ex);
        }
    }
```

**Comment by niqo01** [comment]
Another one: http://crashes.to/s/dba64fa4376

**Comment by Lir10** [comment]
@niqo01 @mcharmas 

Exactly, got the same crashes alot!

`!emitter.isDisposed()` is the solution.

**Comment by mcharmas** [comment]
Should be fixed in 2.1. Propagating to maven central. Should be available in few hours.
